sdks/python/apache_beam/runners/experimental/python_rpc_direct/client.py

```diff
@@ -0,0 +1,123 @@
+#
```

Comment:
All other runners use that pattern for example dataflow_runner, direct_runner, fn_api_runner. And this is a runner.